src/app/skill.service.ts

```diff
@@ -18,8 +18,8 @@ export class SkillService {
   private profileSkills = new BehaviorSubject<Skill>(null);
   profileSkills$ = this.profileSkills.asObservable();
 
-  private url1 = 'http://localhost:8765/interview-service/skills/allSkills';
-  private url2 = 'http://localhost:8765/interview-service/skills/insertSkill';
+  private url1 = 'http://localhost:8765/interview-service/skills';
```

Comment:
Can we also rename this to baseUrl to properly reflect what this variable represents?